packages/office-ui-fabric-react/src/components/SearchBox/SearchBox.styles.tsx

```diff
@@ -142,6 +142,16 @@ export function getStyles(props: ISearchBoxStyleProps): ISearchBoxStyles {
         selectors: {
           '::-ms-clear': {
             display: 'none'
+          },
+          '::placeholder': {
+            color: palette.neutralSecondary,
+            opacity: 1
+          },
+          ':-ms-input-placeholder': {
+            color: palette.neutralSecondary
+          },
+          '::-ms-input-placeholder': {
```

Comment:
are both syntaxes necessary?

Reply:
Apparently browser versioning and placeholder selectors get hairy, but it looks like using just the two will work on at at least all my browser versions.